### DISCUSSION TITLE
My container takes A LOT of RAM?

### DISCUSSION THREAD

**Discussion by TangJames** [thijsvanloef/palworld-server-docker#242] [Q&A] [answered]
Just curious if this is normal, I've allocated 25GB to docker desktop itself. My one server takes up around 3.5GB just idle and will often max out the 25GB. The maxing out part I believe may just be memory leak because server restart fixes that, but does this seem normal?

**Comment by thijsvanloef** [comment]
Sadly there is a memory leak in the server requiring you to reboot your server every ~24h or so.
https://github.com/thijsvanloef/palworld-server-docker/issues/8

**Comment by zbellerose** [comment, marked as answer]
I have 64gb of ram on my server and palworld takes up 30+ if I leave it running without restarting. This is a mem leak with palworld and not related to this image.
